src/modules/75-cf/pages/onboarding/views/SelectEnvironmentView.tsx

```diff
@@ -32,19 +32,27 @@ export const SelectEnvironmentView: React.FC<SelectEnvironmentViewProps> = props
   const { getString } = useStrings()
   const { showError } = useToaster()
   const [environmentCreated, setEnvironmentCreated] = useState(false)
+  const [dialogOpen, setDialogOpen] = useState(false)
+  const [createEnvName, setCreateEnvName] = useState<string>('')
   const [environment, setEnvironment] = useState<EnvironmentResponseDTO>()
   const [apiKey, setApiKey] = useState<SelectEnvironmentViewProps['apiKey']>()
   const { trackEvent } = useTelemetry()
   const { EnvironmentSelect, loading, error, refetch, environments } = useEnvironmentSelectV2({
+    allowCreatingNewItems: true,
     onChange: (_value, _environment, userEvent) => {
-      trackEvent(FeatureActions.EnvSelect, {
-        category: Category.FEATUREFLAG,
-        environment: _environment
-      })
-      setEnvironment(_environment)
-      props.setEnvironmentIdentifier(_environment.identifier)
-      if (userEvent) {
-        props.setApiKey(undefined)
+      if (!_environment) {
+        setCreateEnvName(`${_value.label}`)
```

Comment:
Is this to cast to a string, or can it be simplified?
```suggestion
        setCreateEnvName(_value.label)
```

Reply:
Will check

Reply:
I'll make this change - I originally put it in a template string due to some TS error it was giving at the time which seems to be gone now